ShawMedallion/src/main/java/org/usfirst/frc2450/ShawMedallion/OI.java

```diff
@@ -105,10 +105,9 @@ public OI() {
         SmartDashboard.putData("PistonUp", new PistonUp());
         SmartDashboard.putData("PistonDown", new PistonDown());
         SmartDashboard.putData("ApproachTarget", new ApproachTarget());
-        SmartDashboard.putData("LineUpRight", new LineUpRight());
         SmartDashboard.putData("TargetSeek", new TargetSeek());
         SmartDashboard.putData("EleUP", new EleUP());
-        SmartDashboard.putData("TargetAlign", new TargetAlign());
+        SmartDashboard.putData("TargetAlign", new TargetAlign("right"));
```

Comment:
Should change text "TargetAlign" to "TargetAlignRight" to reflect what it's doing.